src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -267,6 +271,28 @@ def create(dataset, annotations=None, feature=None, model='darknet-yolo',
         'mlmodel_path': pretrained_model_path,
     }
 
+    if batch_size < 1:
+        batch_size = 32  # Default if not user-specified
```

Comment:
The C++ implementation defines the default itself. If the user does not specify a value, pass no batch size to use the heuristic for computing batch size defined in C++